Fix Clone Action of Configuration is out of place

gally47 opened the pull request:
## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
fixes #13874

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

ousmaneo reviewed (changes requested):
Looks good, I think we can take the chance to move the file to `Typescript`.

ousmaneo commented on changelog/unreleased/issue-13874.toml:
```diff
@@ -0,0 +1,5 @@
+type = "fixed"
+message = "Fix Clone Action of Configuration is out of place"
```
```suggestion
message = "Fix Clone Action style on Collectors configurations list "
```

mnin reviewed (approved):
LGTM

mnin reviewed (approved):
LGTM

please use `git merge` instead of `git rebase` after first review was started, to give the reviewer to track the newest changes

gally47 commented:
> Looks good, I think we can take the chance to move the file to `Typescript`.

@ousmaneo is on vacation but he originally requested changes. @mnin reviewed the requested changes and approved them. So, I'll (try to) dismiss @ousmaneo 's review as I addressed the comments, so I can merge this PR. Leaving this comment here to clarify why I dismiss a review.
